ssn/chapters/Actuation.html

```diff
@@ -161,7 +169,8 @@ <h5><a href="#SOSAActuationCollection">sosa:ActuationCollection</a></h5>
             Collection (set) of one or more <a href="#SOSAActuation">Actuations</a>
             or <a href="#SOSAActuationCollection">ActuationCollections</a>, i.e., collections may be nested.
             <br><br>
-            The following <a href="#SOSAActuation">Actuation</a> properties may appear in an <a href="#SOSAActuationCollection">ActuationCollection</a>:
+            The following <a href="#SOSAActuation">Actuation</a> properties may appear in an <a
+                href="#SOSAActuationCollection">ActuationCollection</a>:
             <a href="#SOSAhasFeatureOfInterest">hasFeatureOfInterest</a>
             , <a href="#SOSAusedProcedure">usedProcedure</a>
             , <a href="#SOSAmadeByActuator">madeByActuator</a>
```

Comment:
Similar should be applied throughout. ` ,` (space-comma) and ` .` (space-stop) should (almost? I cannot quickly think of any counter examples.) never appear in streaming text.
```suggestion
            <a href="#SOSAhasFeatureOfInterest">hasFeatureOfInterest</a>,
            <a href="#SOSAusedProcedure">usedProcedure</a>,
            <a href="#SOSAmadeByActuator">madeByActuator</a>,
```

Reply:
Not specific to this PR. 
Please address HTML style and document formatting concerns in a separate issue.

Reply:
See #326